lib/ansible/modules/cloud/amazon/iam_group.py

```diff
@@ -401,11 +410,11 @@ def main():
 
     argument_spec = dict(
         name=dict(required=True),
-        managed_policy=dict(default=[], type='list'),
+        managed_policies=dict(default=[], type='list', aliases=['managed_policy']),
```

Comment:
I think such changes need a changelog fragment (and can't be backported). It's probably better to separate them from the rest of the PR, so that the rest can be backported.

Reply:
Ok, will come out on the next push